### DISCUSSION TITLE
Automatic cookie setup

### DISCUSSION THREAD

**Discussion by codelauncher444** [acheong08/EdgeGPT#538] [Q&A]
Hi cokmunity. How can automatic cookie set up performed? Can you please help me . Feel free to share a code!

**Comment by zhaoqf-cq** [comment]
使用 selenium 打开浏览器, 打开 http://www.bing.com/chat 的地址, 然后使用 cookies = driver.get_cookies() 获取到 cookie, 然后组装json, 你懂的
